cxx/assets/test_files/test_crosscat_categorical.schema

```diff
@@ -0,0 +1,4 @@
+col1 ~ stringcat[strings="a:b:c:d",delim=:](id)
```

Comment:
In addition to the "has_" issue, there also appears to be an off by one error.  Here, "col1" is the one that has a/b/c/d values, but it is "has_col0" in the .obs file that has those values.  

Similarly for the other relations.

Reply:
Fixed as well.